app/components/channel_drawer/channels_list/filtered_list/index.js

```diff
@@ -25,13 +35,17 @@ function mapStateToProps(state, ownProps) {
         profiles = getProfilesInCurrentTeam(state);
     }
 
+    const searchOrder = Config.SidebarSearchOrder ? Config.SidebarSearchOrder : DEFAULT_SEARCH_ORDER;
```

Comment:
can we rename `SidebarSearchOrder` to `DrawerSearchOrder` or `ChannelSearchOrder` ? Sidebar is not used anywhere in the app to refer to the drawer